packages/java/tests/spring/react-grid-test/src/test/java/dev/hilla/test/reactgrid/ReadOnlyGridOrFilterIT.java

```diff
@@ -0,0 +1,33 @@
+package dev.hilla.test.reactgrid;
+
+import org.junit.Test;
+
+import com.vaadin.flow.component.textfield.testbench.TextFieldElement;
+
+public class ReadOnlyGridOrFilterIT extends AbstractGridTest {
```

Comment:
ITs currently don't cover the and filter and the equals matcher. Would be nice to add them, or alternatively write unit tests for the spec conversion.

Reply:
Added unit tests